src/redux/explorer.js

```diff
@@ -536,7 +538,7 @@ const listenOnAddressMessages = socket => dispatch => {
         '😬 Cancelling fallback data provider listener. Zerion is good!'
       );
       dispatch(disableFallbackIfNeeded());
-      dispatch(explorerInitL2());
+      dispatch(explorerInitL2(Network.optimism));
```

Comment:
yo I think you want NetworkTypes here (not the enum Network) assuming it doesn't play nice mixing ts and js here